pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -459,6 +460,46 @@ func (cloud *CloudProvider) InsertDisk(ctx context.Context, project string, volK
 	}
 }
 
+func (cloud *CloudProvider) UpdateDisk(ctx context.Context, project string, volKey *meta.Key, existingDisk *CloudDisk, params common.ModifyVolumeParameters) error {
+	klog.V(5).Infof("Updating disk %v", volKey)
+	// hyperdisks are zonal disks
```

Comment:
What about HdHA (balanced)? I think it would be worth checking that the "volKey.Type()" is Zonal

And issuing an error if it is not Zonal

Reply:
Done